app/paths.ts

```diff
@@ -50,11 +50,15 @@ const paths = {
     questions: '/eligibility',
     eligible: '/eligible',
 
-    // Viewers endppoints
-    documentViewer: '/view/document',
-    viewHomeOfficeDocuments: '/view/home-office-documents',
-    viewAppealDetails: '/appeal-details',
-    viewReasonsForAppeal: '/appeal-reasons',
+    // Viewers endpoints
+    detailsViewers: {
+      document: '/view/document',
+      homeOfficeDocuments: '/view/home-office-documents',
+      appealDetails: '/appeal-details',
+      reasonsForAppeal: '/appeal-reasons',
+      timeExtension: '/view/time-extension',
+      timeExtensionDecision: '/view/time-extension-decision'
+    },
 
```

Comment:
I wouldnt use more than one level object, otherwise journey check middleware would need to flatten the object to pick all the paths in common, also having multiple levels is not adding any value imo

Reply:
I've added these as these are stateless and can't be accessed at any state so the middleware doesn't apply, they were also all related to one thing so I thought it'll be better to keep everything on the same object following the same conventions as  Ask for more time.